lib/ffi/task_storage.go

```diff
@@ -160,10 +160,14 @@ func (t *TaskStorage) Claim(taskID int) (func() error, error) {
 	}()
 
 	for _, sectorRef := range sectorRefs {
-		if err := t.sc.Sectors.sindex.StorageLock(lkctx, sectorRef.ID(), storiface.FTNone, requestedTypes); err != nil {
+		ok, err := t.sc.Sectors.sindex.StorageTryLock(lkctx, sectorRef.ID(), storiface.FTNone, requestedTypes)
```

Comment:
I'd honestly just entirely get rid of the sector storage lock stuff, it's basically entirely redundant with harmonytask managing pipelines